modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -77,13 +77,13 @@ If you use Anypoint Connector for File (File Connector) and share part of the pa
 ----
 
 === HTTP and HTTPS
-For HTTP and HTTPS, use the following placeholders, depending on the deployment model:
 
-* `${http.port}` +
-When deploying to CloudHub or Hybrid using HTTP (port=8081) or HTTPS (port=8082)
+HTTP and HTTPS property placeholders are reserved in CloudHub deployments. Use of the same placeholders is recommended for on-premises deployments.
 
-* `${http.private.port}` +
-When deploying to CloudHub using HTTP (port=8091) or HTTPS (port=8092) and using a Dedicated Load Balancer
+* In CloudHub, `${http.port}` maps to the setting `port=8081`, and `${https.port}` maps to `port=8082`.
+* For internal ports to HTTP services in CloudHub deployments, `${http.private.port}` maps to the setting `port=8091`, and `${https.private.port}` maps to `port=8092`.
```

Comment:
Similar to the previous item. Those internap ports are only relevant because it is the default port used by the dedicated load balancers. Can we clarify that or reference another doc with that information?

Reply:
```suggestion
* For internal ports to HTTP services in CloudHub deployments, `${http.private.port}` maps to the setting `port=8091`, and `${https.private.port}` maps to `port=8092`. The ports are the defaults used by the shared load balancer in CloudHub. 
```

Reply:
see above